docs/source/markdown/podman-container-clone.1.md

```diff
@@ -0,0 +1,165 @@
+% podman-container-clone(1)
+
+## NAME
+podman\-container\-clone - Creates a copy of an existing container
+
+## SYNOPSIS
+**podman container clone** [*options*] *container*
+
+## DESCRIPTION
+**podman container clone** creates a copy of a container, recreating all namespaces using the original options when the container was created allowing users to manipulate the original configuration.
+
+## OPTIONS
+
+#### **--name**
+
+Set a custom name for the cloned container. The default if not specified is of the syntax: **<ORIGINAL_NAME>-clone**
+
+#### **--destroy**
+
+Remove the original container that we are cloning once used to mimic the configuration
+
+#### **--cpus**
+
+Set a number of CPUs for the container that overrides the original containers CPU limits. If none are specified, the original container's Nano CPUs are used.
+
+This is shorthand
+for **--cpu-period** and **--cpu-quota**, so you may only set either
```

Comment:
```suggestion
for **--cpu-period** and **--cpu-quota**, so you may only set
```

Reply:
should i also delete `so only` here @TomSweeneyRedHat ?

Reply:
ah nevermind I see the other edits below